homeassistant/components/radarr/coordinator.py

```diff
@@ -105,7 +105,10 @@ class HealthDataUpdateCoordinator(RadarrDataUpdateCoordinator[list[Health]]):
 
     async def _fetch_data(self) -> list[Health]:
         """Fetch the health data."""
-        return await self.api_client.async_get_failed_health_checks()
+        health = await self.api_client.async_get_failed_health_checks()
+        if isinstance(health, Health):
+            health = [health]
```

Comment:
We can avoid the assignment here

```suggestion
            return [health]
```